backend/canisters/user_index/impl/src/model/external_achievements.rs

```diff
@@ -24,11 +24,12 @@ pub struct ExternalAchievementInternal {
 }
 
 impl ExternalAchievements {
+    // This will either a register a new external achievement or update an existing entry with the same id
```

Comment:
Typo

Reply:
I had fixed in VS code and not saved the file!